islands/package.json

```diff
@@ -12,7 +12,7 @@
     "yarpm": "^1.2.0"
   },
   "scripts": {
-    "start": "cd islands-project/islands-app && yarpm start",
+    "start": "cd islands-project/islands-app && yarpm start --",
```

Comment:
Note, this addition was necessary to enable extra `npm run ...` args to work, i.e.:

`npm run start -- --account <account name> --config <config path> --ssl`

Though that change is _not_ needed for this to work (likely only needed for `yarpm`):

`npm run deploy -- --account <account name> --config <config path>`